### DISCUSSION TITLE
METplus Coordinated 5.1.0-beta1 on HERA

### DISCUSSION THREAD

**Discussion by JiayiPeng-NOAA** [dtcenter/METplus#2089] [Installation] [answered]
Hi MetPlus Team,
Do I need to load Python 3.10 to run METplus 5.1.0-beta1 on Hera?
For the previous MetPlus version, I load anaconda/latest (see below):
-----------------------------
module use -a /contrib/anaconda/modulefiles
module load intel/${intel_ver}
module load anaconda/latest
module use -a /contrib/met/modulefiles/
module load met/${met_ver}
module use /contrib/METplus/modulefiles
module load metplus/${metplus_ver}
------------------------------
Thanks,
Jiayi
EMC/NCEP/NOAA

**Comment by jprestop** [comment, marked as answer]
Hi @JiayiPeng-NOAA.  

In loading METplus-5.1.0-beta1 on Hera, you will not need to load anaconda/latest, but rather you can to follow the instructions on the Existing Builds page for loading METplus-5.1.0-beta1:
```
module load intel/2022.1.2
module use /contrib/METplus/modulefiles
module load metplus/5.1.0-beta1
```

In the loading the modulefile for METplus, a version of Python 3.10 is added to your path:
```
prepend-path PATH /scratch1/BMC/dtc/miniconda/miniconda3/envs/metplus_v5.1_py3.10/bin
```

We built MET-11.1.0-beta1 using that same version of Python, whereas previously we used the anaconda/latest (Python3.7) to build MET.
